### PR TITLE
Add metrics labels regex rules

### PR DESCRIPTION
**Pull request by yaron2**
Closes https://github.com/dapr/dapr/issues/4133.

### CODE REVIEW

File: pkg/diagnostics/utils/metrics_utils.go [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021 The Dapr Authors
+Copyright 2022 The Dapr Authors
```
Comment (ItalyPaleAle) [inline pkg/diagnostics/utils/metrics_utils.go:2]: 3? Since you're updating this :)

**Review by ItalyPaleAle — APPROVED** [review]
Please don't forget to create a docs issue

File: pkg/diagnostics/utils/metrics_utils.go (hunk 1 above)
Comment (yaron2) [reply pkg/diagnostics/utils/metrics_utils.go:2]: OMG my brain didn't register it's 2023 yet.

**Comment by yaron2** [comment]
Docs tracked here: https://github.com/dapr/docs/issues/3046.

**Review by ItalyPaleAle — APPROVED** [review]

**Pull request merged**